helm/hydra/templates/_helpers.tpl

```diff
@@ -71,7 +71,7 @@ readinessProbe:
 - name: SYSTEM_SECRET
   valueFrom:
     secretKeyRef:
-      name: hydra-secret
+      name: {{ required "Hydra Helm chart requires `systemSecretName` to be specifed and persisted in Kubernetes. See README for more information" .Values.systemSecretName }}
```

Comment:
The `values.yaml` key wasn't used yet previously; now it is. Example of the results (I'm using `helm template` but it could be `helm install`):

Error:
```
helm template \
    --name hydra \
    --namespace hydra ./ > .rendered.yaml

Error: render error in "hydra/templates/deployment.yaml": template: hydra/templates/deployment.yaml:39:14:
  executing "hydra/templates/deployment.yaml" at <include "hydra.secre...>: 
    error calling include: template: hydra/templates/_helpers.tpl:74:15: 
      executing "hydra.secrets" at <required "Hydra requ...>: 
        error calling required: Hydra Helm chart requires `systemSecretName` to be specifed and persisted in Kubernetes. See README for more information.

(this output was wrapped for display on GitHub; original line count: 2)
```

OK:
```
helm template \
  --name hydra \
  --namespace hydra \
  --set systemSecretName=hydra-system-secret ./ > .rendered.yaml
```

Reply:
Awesome!